modules/core/migrations/20240602202521-initial.sql

```diff
@@ -3,12 +3,12 @@ BEGIN;
 
 CREATE TABLE uploads
 (
-    id         SERIAL PRIMARY KEY,
-    name       VARCHAR(255) NOT NULL,
-    url        VARCHAR(255) NOT NULL,
---     uploader_id INT          REFERENCES users (id) ON DELETE SET NULL,
-    mimetype   VARCHAR(255) NOT NULL,
-    size       INT          NOT NULL,
+    id         VARCHAR(255) PRIMARY KEY,
+    url        VARCHAR(1024) NOT NULL   DEFAULT '',
```

Comment:
Are u sure this even works? I think VARCHAR support 255 max, and if you wanna go beyond that you use TEXT

Reply:
Yeah, I think it goes up to ~65K characters

Reply:
You're right. My knowledge of postgres is way outdated turns out)